yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/command/QueryConfig.java

```diff
@@ -154,7 +153,7 @@ private static String appendWithContinuationIfPresent(@Nonnull String queryStrin
         if (!currentQuery.isEmpty() && currentQuery.charAt(currentQuery.length() - 1) == ';') {
             currentQuery = currentQuery.substring(0, currentQuery.length() - 1);
         }
-        if (continuation instanceof ContinuationImpl) {
+        if (continuation != null) {
```

Comment:
Should we be adding any sort of assertions to the yaml test framework about the reason for the continuation?
Should we assert that we never get back the beginning continuation?
I don't think the yaml tests have a good way to cause `TRANSACTION_LIMIT_REACHED`, but the others should be testable in an integrated way, which seems worthwhile.

Reply:
Created a ticket